sonar-db/src/main/java/org/sonar/db/permission/PermissionTemplateDao.java

```diff
@@ -181,22 +180,33 @@ public PermissionTemplateDto selectPermissionTemplate(String templateKey) {
     }
   }
 
-  public PermissionTemplateDto insertPermissionTemplate(String templateName, @Nullable String description, @Nullable String keyPattern) {
+  /**
+   * @deprecated since 5.2 use {@link #insert(DbSession, PermissionTemplateDto)}
+   */
+  @Deprecated
+  public PermissionTemplateDto insert(String templateName, @Nullable String description, @Nullable String projectPattern) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)